### ISSUE TITLE
Make a new release, finally?

### ISSUE THREAD

**Issue by ghost** [oguzhaninan/Stacer#464]
Newest release is from 2019 and I see that there was some development so would it be possible to make a new release?

**Comment by alexmyczko** [comment]
Yes maybe, 1.1.1 (or whatever you like, that's bigger than 1.1.0). Now is the best time if you want the package be shipped with Ubuntu 22.04 LTS (deadline for debian archive sync is around mid Feb 2022).

Debian will freeze mid Jan 2023 for a new release, a new version would be great...

**Comment by Anutrix** [comment]
@maintainers
